### PR TITLE
deploy: bump docs to 2024.07.02

### PR DESCRIPTION
**Pull request by siers**

### CODE REVIEW

File: updated-versions.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+
+set -eu
+
+getToolsReleases() {
+  curl -s "https://api.github.com/repos/dasch-swiss/dsp-tools/releases"
+}
+
+getDeployReleases() {
+  curl -s "https://raw.githubusercontent.com/dasch-swiss/ops-deploy/main/versions/RELEASE.json?token=GHSAT0AAAAAACNWGVNZCBSXY52KPHVWAXAEZVTIPCQ"
```
Comment (jnussbaum) [inline updated-versions.sh:10]: This repo is public, so I guess it's a security concern to have this token here!

Comment (siers) [reply updated-versions.sh:10]: Ah, so that's why there's a token, – it's private. 🤦

Comment (jnussbaum) [reply updated-versions.sh:10]: Yes, ops-deploy is private, and contains highly sensitive data, while dsp-docs is public

Comment (siers) [reply updated-versions.sh]: Token already expired, I think it was a once-off token, removed the curl entirely so it automatically fails.

**Pull request merged**